Package@swift-5.7.swift

```diff
@@ -0,0 +1,51 @@
+// swift-tools-version: 5.6
```

Comment:
should this be 5.7 here?

Reply:
This can be 5.6 since we don’t use any 5.7 toolchain feature in here. The name of the file is the important part.